app/resources/forms.js

```diff
@@ -329,3 +329,19 @@ function validate_fields() {
 
   return true;
 }
+
+// Expand the form to see the detail form
+function expand_forms(expandSection) {
+  jQuery(document).ready(function() {
+    var expandPart = jQuery(expandSection).children("div.expand-part");
```

Comment:
It looks it's better to use find() than children(). children() only selects direct children of the element, while find() selects all its descendants. This expandPart is currently a direct children, but it may not be in the future.